tests/e2e/sample-applications/mysql-persistent/mysql-persistent-csi-aws-template.yaml

```diff
@@ -155,7 +155,7 @@ items:
         spec:
           containers:
           - name: todolist
-            image:  quay.io/rhn_engineering_whayutin/todolist-mariadb-go:latest
+            image: quay.io/mferrato/todolist-mariadb-go:v1
```

Comment:
What's happening here?

Reply:
Wesley's image is amd64 only. I made a PR to the mig-demo-app repo https://github.com/konveyor/mig-demo-apps/pull/74 to add the code enabling multiarch builds for the mariadb-based app. We discused that with @weshayutin as a temporary solution until we have a place to store the multiarch manifest. But I'd rather store it somewhere else, if you know of a better place.

Reply:
Correct.. I've just opened an issue on OADP to better describe what needs to be done here and why. https://github.com/openshift/oadp-operator/issues/666

Pretty crazy re: the issue #   
![7c0696e0-5c7b-48dc-a500-69be86939db9 e6557d1ec7732d827674187696a95a86](https://user-images.githubusercontent.com/138787/166692587-4495101d-7ad1-4412-bee5-7d3223f4ff9b.jpeg)

Reply:
So konveyor quay org would be one